front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -321,6 +329,9 @@ export default function AssistantBuilder({
         if (slackDataSource) {
           await mutateSlackChannels();
         }
+
+        await mutateAgentConfigurations();
```

Comment:
note (unrelated):

If we don't need to "wait", I suggest changing to `void mutateAgentConfigurations()` (same for the slack thing above).